evalml/tests/conftest.py

```diff
@@ -221,7 +221,7 @@ def X_y_binary():
     return X, y
 
 
-@pytest.fixture
```

Comment:
I did this to try and jibe with the new Sequential engine fixture to cut down on runtime and prevent the SequentialEngine from running more than once by using the cached results from the fixture.

Reply:
Sweet - did it work?